contracts/transfer/circuits/src/tests.rs

```diff
@@ -0,0 +1 @@
+pub mod leaf;
```

Comment:
Should this file be removed? Is a leftover?

Reply:
This is used only by the tests and is required since `Note` don't implement `PoseidonLeaf` by default

Reply:
Fixed